### PR TITLE
Added the UI for addRecipe form.

### PR DESCRIPTION
**Pull request by siddhantk232**
This PR introduces the UI required to create a new recipe in the recipe-app.

### CODE REVIEW

File: client/src/views/Forms/RecipeForm/index.js [hunk 1]
```diff
@@ -1,100 +1,151 @@
...
+   )
+}
+
+function AddServings({ close }) {
+   return (
+      <div style={{ padding: '30px' }}>
+         <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+            <div style={{ display: 'flex', alignItems: 'center' }}>
+               <TextButton onClick={() => close(1)} type='ghost'>
+                  X
```
Comment (thesanjeevsharma) [inline client/src/views/Forms/RecipeForm/index.js]: Use an Icon here, not 'X'!

File: client/src/views/Forms/RecipeForm/index.js [hunk 2]
```diff
@@ -1,100 +1,151 @@
...
+                     setrecipeName(e.target.value)
+                  }}
+               />
+            </div>
 
-				<div style={{ width: '100%', textAlign: 'right' }}>
-					<TextButton type="ghost" style={{ margin: '0px 10px' }}>
-						open in editor
-					</TextButton>
+            <div style={{ width: '100%', textAlign: 'right' }}>
```
Comment (thesanjeevsharma) [inline client/src/views/Forms/RecipeForm/index.js]: Avoid doing these! Use 'styled-components'.

Comment (siddhantk232) [reply client/src/views/Forms/RecipeForm/index.js]: I will do it in the coming PRs that's part of my refactor.

File: client/src/views/Forms/RecipeForm/index.js (hunk 1 above)
Comment (siddhantk232) [reply client/src/views/Forms/RecipeForm/index.js]: ok but what if I do this in the next PR

File: client/src/store/tabs/index.js [hunk 3]
```diff
@@ -5,11 +5,22 @@ const Context = React.createContext()
...
    forms: [],
-   current: {},
+   current: {}
 }
 
 const reducers = (state, { type, payload }) => {
    switch (type) {
+      case 'SET_RECIPE_TITLE': {
+         const newState = { ...state }
+         if (payload.title.length > 0) {
```
Comment (thesanjeevsharma) [inline client/src/store/tabs/index.js]: 1. Make it a generic action, not just limited to the recipe. Even I'll use it for my ingredient form.
2. Check the title length before dispatching the action. Actions should perform the task only, nothing else.

**Pull request closed without merging**